assets/product-form.js

```diff
@@ -6,16 +6,17 @@ if (!customElements.get('product-form')) {
       this.form = this.querySelector('form');
       this.form.querySelector('[name=id]').disabled = false;
       this.form.addEventListener('submit', this.onSubmitHandler.bind(this));
-      this.cartNotification = document.querySelector('cart-notification');
+      this.cart = document.querySelector('cart-notification') || document.querySelector('cart-drawer');
       this.submitButton = this.querySelector('[type="submit"]');
+      if (document.querySelector('cart-drawer')) this.submitButton.setAttribute('aria-haspopup', 'dialog');
     }
 
     onSubmitHandler(evt) {
       evt.preventDefault();
       if (this.submitButton.getAttribute('aria-disabled') === 'true') return;
 
       this.handleErrorMessage();
-      this.cartNotification.setActiveElement(document.activeElement);
+      this.cart.setActiveElement(document.activeElement);
```

Comment:
ah, do you want to check this PR then 😬  https://github.com/Shopify/dawn/pull/1638

Reply:
@ludoboludo 
My bad. I just didn't realize that was still the case once I saw the setting.
It's literally in the PR description that it will be tackled in a separate PR. 🤦‍♂️

I've added myself as a reviewer there and I'm ready to approve this one now 👀